corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -44,7 +25,7 @@ function LocationSettingsViewModel() {
 
     this.new_loctype = function() {
         var parent_pk = (_.last(settings.loc_types()) || {}).pk;
-        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, this);
+        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, commtrack_enabled);
```

Comment:
sorry to keep last-minute updating, but having looked closer at how this code works it seems like just adding the `commtrack_enabled` variable to `settings` would be clearer and more consistent with what it looks like the `load` function is intended to do. any reason you didn't do that?

(note that I don't think any of my suggestions have necessarily been blocking - am more just trying to channel some of the tech debt/code review stuff from the summit)